### DISCUSSION TITLE
How to custom ftplugin

### DISCUSSION THREAD

**Discussion by tuanha168** [NvChad/NvChad#1482] [Q&A]
I can't find anyone mentions this, but some plugins need to custom in `nvim/ftplugin/*.lua` file.
Like https://github.com/mfussenegger/nvim-jdtls

So how do i custom it without being erased every time I update nvchad

**Comment by kumkee** [comment]
What I did was creating a `ftplugin` folder under `custom` and symlinking it to `nvim/`
Since `ftplugin` is in [gitignore](https://github.com/NvChad/NvChad/blob/main/.gitignore), it won't be erased. The only reason I put it in `custom` is that I can back it up with the rest of my configuration files.
